src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -18,19 +21,38 @@ import TableHeader from "components/molecules/TableHeader";
 import { UserList } from "components/molecules/UserList";
 import { TablesUserList } from "components/molecules/TablesUserList";
 
+import { BackButton } from "components/atoms/BackButton";
+
 import { TABLES } from "./constants";
 
 import "./ConversationSpace.scss";
 
+export type ConversationSpaceProps = {
+  venue?: WithId<AnyVenue>;
+};
+
 // @debt refactor this to pass in venue as a prop
-export const ConversationSpace: React.FunctionComponent = () => {
-  const venue = useSelector(currentVenueSelectorData);
+export const ConversationSpace: React.FunctionComponent<ConversationSpaceProps> = ({
```

Comment:
```suggestion
export const ConversationSpace: React.FC<ConversationSpaceProps> = ({
```

Reply:
fixed in c31874f2e06bf3af652db14f326f77495653a166